server/server/src/io/log_manager.rs

```diff
@@ -8,6 +8,7 @@ use chrono::Local;
 use platform_dirs::UserDirs;
 
 pub const LOG_FILE_NAME: &str = "server.log";
+const LOG_DIR_PREFIX: &str = "FBugReporter";
 const LOG_DIR: &str = "logs";
```

Comment:
Could you replace `logs` with `server_logs`? We already store client logs in `FBugReporter` directory it could be confusing.

Reply:
Done